docker/1.3.1/py2/mms-entrypoint.py

```diff
@@ -0,0 +1,17 @@
+import shlex
+import subprocess
+import sys
+import os.path
+
+from sagemaker_pytorch_serving_container import serving
```

Comment:
Should this be moved inside the if condition? Keep this consistent with the mms-entrypoint for mxnet.